src/cryptoadvance/specterext/stacktrack/service.py

```diff
@@ -56,13 +57,18 @@ def callback_add_wallettabs(self) -> list[dict[str, str]]:
         }]
 
     def callback_adjust_view_model(self, view_model: WalletsOverviewVm):
+        # This is where we configure view model customizations for StackTrack.
         if type(view_model) == WalletsOverviewVm:
-            # potentially, we could make a reidrect here:
-            # view_model.about_redirect=url_for("spectrum_endpoint.some_enpoint_here")
-            # but we do it small here and only replace a specific component:
-            view_model.tx_table_include  = "stacktrack/overview.jinja"
+            # view_model.header_and_summary_include = "..."
+            # view_model.balance_overview_include = "..."
+
+            # Redirect to our custom StackTrack controller so we can generate a chart.
+            view_model.wallets_overview_redirect = url_for("stacktrack_endpoint.wallets_overview")
+            # Replace the default tx table with one that includes a chart.
+            view_model.tx_table_include = "stacktrack/wallet/overview/overview_chart_and_tx_table.jinja"
```

Comment:
That should imho be deleted here. You cannot do both: Either redirect or template replacements.

Reply:
Perfect, thanks.